lib/galaxy/tools/cache.py

```diff
@@ -25,65 +18,42 @@
 CURRENT_TOOL_CACHE_VERSION = 2
```

Comment:
Should we increase (or reset) this value? We now cache `document` instead of `metadata`.